pkg/process/procutil/process_linux.go

```diff
@@ -212,10 +212,17 @@ func (p *Probe) ProcessesByPID(now time.Time) (map[int32]*Process, error) {
 			continue
 		}
 
-		statusInfo := p.parseStatus(pathForPID)
-		statInfo := p.parseStat(pathForPID, pid, now)
-		memInfoEx := p.parseStatm(pathForPID)
+		var (
+			statusInfo = &statusInfo{}
+			statInfo   = &statInfo{}
+			memInfoEx  = &MemoryInfoExStat{}
+		)
 
+		if collectStats {
```

Comment:
Aren't we still collecting `Stats.OpenFdCount` and `Stats.IOStat` if `collectStats = false`?

Reply:
Fixed! https://github.com/DataDog/datadog-agent/pull/9082/commits/250d2af88df6768f3c67bedd7c858256bd746d90